checkov/openapi/checks/resource/v2/Oauth2OperationObjectPasswordFlow.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+from typing import Any, Union, List
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.common.checks.enums import BlockType
+from checkov.openapi.checks.resource.v2.BaseOpenapiCheckV2 import BaseOpenapiCheckV2
+
+
+class Oauth2OperationObjectPasswordFlow(BaseOpenapiCheckV2):
+    def __init__(self) -> None:
+        id = "CKV_OPENAPI_10"
+        name = "Ensure that operation object does not use 'password' flow in OAuth2 authentication - version 2.0 files"
+        categories = [CheckCategories.API_SECURITY]
+        supported_resources = ["paths"]
+        super().__init__(
+            name=name,
+            id=id,
+            categories=categories,
+            supported_entities=supported_resources,
+            block_type=BlockType.DOCUMENT,
+        )
+
+    def scan_openapi_conf(  # type:ignore[override]
```

Comment:
is the ignore necessary? other checks didnt have it, so maybe it can be removed here also?